### DISCUSSION TITLE
Why clicked on same version from the version switcher the page takes to the landing page

### DISCUSSION THREAD

**Discussion by sunilkn020** [squidfunk/mkdocs-material#6810] [Q&A] [answered]
For the URL: https://opensource.hcltechsw.com/digital-experience/CF216/extend_dx/ddc/#hclsoftware-u-learning-materials, it is already in CF216 and when I select 216 again from the switcher, I get the landing page. I also tried this in other subtopics (selecting the same doc version in the version switcher) and the same behaviour was observed.

**Comment by kamilkrzyskow** [comment, marked as answer]
Could you please not spam? There is another such discussion here from 2 hours ago: 
- https://github.com/squidfunk/mkdocs-material/discussions/6809

There are also 2 reported issues on the matter, both without a minimal example:
- https://github.com/squidfunk/mkdocs-material/issues/6657
- https://github.com/squidfunk/mkdocs-material/issues/6716

The issues were a follow-up after:
- https://github.com/squidfunk/mkdocs-material/discussions/6537

I'll lock the conversation, and you can follow-up in: https://github.com/squidfunk/mkdocs-material/discussions/6809
